app/wama.core/ViewModel/User/UserAccountViewModel.cs

```diff
@@ -6,7 +6,7 @@ namespace WAMA.Core.ViewModel.User
     public abstract class UserAccountViewModel
     {
         public virtual UserAccountType AccountType { get; }
-
+        [Required]
```

Comment:
Why are you making this required? 

I had left this out since a person that doesn't have an SDSU ID will not be able to register.
We still have to come up with a means of generating/handling IDs for that case.
During demos we'll avoid living it empty.

I don't get what you mean by you can't check to make sure fields aren't empty in the back-end.
There's no need for JavaScript; at least not yet.
For example, to make sure the member id is not empty.
In the UserAccountAddNewUser action, you can do a check like this 

if (string.IsNullOrWhitespace(user.MemberId))
{
   SetErrorMessages("Missing member ID");
}